### PR TITLE
Mike | Initial commit

### PR DESCRIPTION
**Pull request by mike-robertson**

### CODE REVIEW

**Comment by mike-robertson** [comment]
I left some stuff off, I will fix. I will also try to put in changes made by gu3st so we can move to one codebase sooner.

**Comment by mike-robertson** [comment]
Contains all of gu3st functionality changes (Primarily the change to using <Link> instead of <a>)

File: actions/actions.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export function fetchData(name, options)
     // that is passed on as the return value of the dispatch method.
     // In this case, we return a promise to wait for.
     // This is not required by thunk middleware, but it is convenient for us.
-    return fetch("http://hchung.westus.cloudapp.azure.com:5000"+getEndpoint(name, options), {credentials: 'include'}).then(response => response.json()).then(json =>
+    return fetch(getEndpoint(name, options)).then(response => response.json()).then(json =>
```
Comment (howardchung) [inline actions/actions.js]: need to pass the API host somehow.  Also need the credentials include to support login

File: actions/metadataActions.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import fetch from 'isomorphic-fetch';
+
+const url = 'http://yasp.co/api/metadata';
```
Comment (howardchung) [inline actions/metadataActions.js]: refactor API host to single location

File: actions/userActions.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import fetch from 'isomorphic-fetch';
+
+const url = 'http://yasp.co/api/players';
```
Comment (howardchung) [inline actions/userActions.js]: same

File: actions/userActions.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import fetch from 'isomorphic-fetch';
```
Comment (howardchung) [inline actions/userActions.js]: call user "player" for consistency

File: npm-debug.log [hunk 5]
```diff
@@ -0,0 +1,50 @@
+0 info it worked if it ends with ok
```
Comment (howardchung) [inline npm-debug.log]: should gitignore this

File: package.json [hunk 6]
```diff
@@ -32,15 +32,28 @@
     "babel-core": "6.7.4",
     "babel-loader": "^6.2.3",
     "babel-polyfill": "^6.7.2",
-    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-es2015-webpack": "^6.4.1",
     "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "coveralls": "2.11.9",
```
Comment (howardchung) [inline package.json]: remove the dependencies you didn't add

File: reducers/gotUser.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import { userActions } from '../actions';
```
Comment (howardchung) [inline reducers/gotUser.js]: user->player

File: webpack.config.js [hunk 8]
```diff
@@ -12,12 +14,18 @@ module.exports = {
...
+      extensions: ['', '.jsx', '.js', '.css', '.json']
+    },
+    devtool: 'source-map',
     module:
     {
         loaders: [
             {
                 test: /\.css$/,
-                loader: "style-loader!css-loader"
+                loader: "style-loader!css-loader?"
```
Comment (howardchung) [inline webpack.config.js]: you could put this back now that's there's no risk of breaking existing code

File: yaspv2.js [hunk 9]
```diff
@@ -33,25 +35,23 @@ import
 }
 from 'react-router-redux';
 // Load CSS
+
 require('./node_modules/font-awesome/css/font-awesome.css');
-//require('./node_modules/dota2-minimap-hero-sprites/assets/stylesheets/dota2minimapheroes.css');
```
Comment (howardchung) [inline yaspv2.js]: we're going to need the sprites back at some point (currently used on teamfight death map)

**Comment by mike-robertson** [comment]
working on all these changes and also doing a little more. Need to add in sourcemapping to webpack as well.

File: components/Logo/Logo.jsx [hunk 10]
```diff
@@ -5,11 +5,11 @@ export default () => {
     <div className="navbar-header">
       <a href="/" className="navbar-brand">
          <strong className="theme-blue">YASP</strong>
-         <a data-toggle="collapse" data-target=".navbar-collapse" className="navbar-toggle">
+         <div data-toggle="collapse" data-target=".navbar-collapse" className="navbar-toggle">
```
Comment (gu3st) [inline components/Logo/Logo.jsx]: So, this is actually supposed to look like 
image

I realized (from the live site) that the collapse stuff is for mobile, for toggling the menu on narrow screens. It shouldn't be a descendent of the main navbar brand, as it's floated to the right side on narrow displays.

**Comment by howardchung** [comment]
Couple of minor things, but we can clean it up later

**Pull request merged**